src/convertToObject.js

```diff
@@ -13,6 +13,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const res = {};
+
+  sourceString.split(';'
+  ).map(
+    (a) => a.split(':'
+    ).map(
+      (c) => c.trim())
+  ).filter(
+    (a) => a[0] !== '' && a[0] !== undefined
+  ).map(
+    (a) => {
+      res[a[0]] = a[1];
+    });
```

Comment:
```suggestion
  sourceString.split(';')
    .map(
      (a) => a.split(':').map((c) => c.trim())
    ).filter(
      (a) => a[0] !== '' && a[0] !== undefined
    ).forEach(
      (a) => { res[a[0]] = a[1]; }
    );
```
Зачем в последнем случае использовать map, если ты ничего из него не возвращаешь?
Форматирование трудно читаемое
Что такое a и с? Давай переменным понятные имена, чтобы я знал с чем именно я сейчас работаю